pages/about-us/newsletter/2012/02/en.md

```diff
@@ -1,6 +1,6 @@
 @title = 'February'
 
-h2. Notes from the Help Desk: Tech advice on email clients
+## Notes from the Help Desk: Tech advice on email clients
 
 bq.. Dear Riseup,
```

Comment:
This `bq..` doesn't render things right in markdown

Reply:
done